internal/receiver/databricksreceiver/metadata.yaml

```diff
@@ -215,181 +221,212 @@ metrics:
   databricks.spark.dag_scheduler.jobs.active:
     enabled: true
     description: n/a
+    unit: "{jobs}"
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.dag_scheduler.jobs.all:
     enabled: true
     description: n/a
+    unit: "{jobs}"
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.dag_scheduler.stages.failed:
     enabled: true
     description: n/a
+    unit: "{stages}"
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.dag_scheduler.stages.running:
     enabled: true
     description: n/a
+    unit: "{stages}"
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.dag_scheduler.stages.waiting:
     enabled: true
     description: n/a
+    unit: "{stages}"
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
+  # https://spark.apache.org/docs/latest/monitoring.html
   databricks.spark.executor_metrics.direct_pool.memory:
     enabled: true
-    description: n/a
+    description: Peak memory that the JVM is using for direct buffer pool
+    unit: By
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
+  # https://spark.apache.org/docs/latest/monitoring.html
   databricks.spark.executor_metrics.jvm.heap.memory:
     enabled: true
-    description: n/a
+    description: Peak memory usage of the heap that is used for object allocation. See https://spark.apache.org/docs/latest/monitoring.html
+    unit: By
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
+  # https://spark.apache.org/docs/latest/monitoring.html
   databricks.spark.executor_metrics.jvm.off_heap.memory:
     enabled: true
-    description: n/a
+    description: Peak memory usage of non-heap memory that is used by the Java virtual machine. See https://spark.apache.org/docs/latest/monitoring.html
+    unit: By
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.executor_metrics.major_gc.count:
     enabled: true
-    description: n/a
+    description: Total major GC count. For example, the garbage collector is one of MarkSweepCompact, PS MarkSweep, ConcurrentMarkSweep, G1 Old Generation and so on.
+    unit: "{gc}"
```

Comment:
fwiw I see in the spark receiver they use `{gc_operation}`. I'd think the plural form of that could preferable but think `{gc}` is ok too since it's shorter.

Reply:
```suggestion
    unit: "{gc_operations}"
```